withings_api/common.py

```diff
@@ -840,6 +908,56 @@ def get_measure_value(
     )
 
 
+def new_heart_get_response(data: dict) -> HeartGetResponse:
+    """Create GetSleepResponse from json."""
+    return HeartGetResponse(
+        signal=tuple(sample for sample in data.get("signal", [])),
```

Comment:
Default value is a mutable empty array `[]`. Change to immutable empty tuple `()`.